tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3247,6 +3250,42 @@ def test_transaction_user_misery(self) -> None:
             },
         }
 
+    def test_support_columns_from_tables(self) -> None:
+        apdex = "apdex(300)"
+        query = "event.type:transaction transaction.op:pageload some_tag:#foo"
+        # The fields include function and non-function columns. transaction will be used as a group-by
+        params = {"field": [apdex, "transaction"], "query": query, "yAxis": apdex}
+        specs = self._create_specs(params)
+        days_ago = before_now(days=1).replace(hour=10, minute=0, second=0, microsecond=0)
+        for spec in specs:
+            for hour in range(0, 5):
+                self.store_on_demand_metric(
+                    1,
+                    spec=spec,
+                    additional_tags={"satisfaction": "tolerable"},
```

Comment:
If I adjust the values of `additional_tags` I will see the apdex score change as expected.